raiden/ui/app.py

```diff
@@ -12,13 +12,10 @@
 import json
 from eth_utils import encode_hex
 
-from raiden.network.transport.matrix.utils import get_available_servers_from_config, server_is_available
-from raiden.storage import serialize, sqlite
+from raiden.network.transport.matrix.layer import MatrixLayer
```

Comment:
lots of unused imports removed in this file due to taking the `_setup_matrix` method out of here